pkg/bench/rttanalysis/testdata/benchmark_expectations

```diff
@@ -1,65 +1,65 @@
 exp,benchmark
-10,AlterRole/alter_role_with_1_option
-13,AlterRole/alter_role_with_2_options
-17,AlterRole/alter_role_with_3_options
-9,AlterTableAddCheckConstraint/alter_table_add_1_check_constraint
-9,AlterTableAddCheckConstraint/alter_table_add_2_check_constraints
-9,AlterTableAddCheckConstraint/alter_table_add_3_check_constraints
-9,AlterTableAddColumn/alter_table_add_1_column
-9,AlterTableAddColumn/alter_table_add_2_columns
-9,AlterTableAddColumn/alter_table_add_3_columns
-11,AlterTableAddForeignKey/alter_table_add_1_foreign_key
-13,AlterTableAddForeignKey/alter_table_add_2_foreign_keys
-15,AlterTableAddForeignKey/alter_table_add_3_foreign_keys
-11,AlterTableAddForeignKey/alter_table_add_foreign_key_with_3_columns
+14,AlterRole/alter_role_with_1_option
```

Comment:
@fqazi can I also bug you to lgtm these changes? I looked at one of the jaeger json files and the additional roundtrips are the delete and write to the jobs info table.